src/view/trustlines-101.njk

```diff
@@ -5,6 +5,9 @@
 <section class="section">
   <div class="container">
     <div class="content">
+      <div class="columns">
```

Comment:
The indentation of the content does not match anymore by this new `<div>`. Simply indent it one level more. Unfortunately I can't find a formatting tool for _Nunjucks_ templates which could solve such issues automatically. :expressionless:

Reply:
Resolved